CONTRIBUTING.md

```diff
@@ -20,8 +20,37 @@ You can open a new issue with this [issue form](https://github.com/novuhq/novu/i
 
 ### Requirements
 
-- Node.js version v16.15.1
-- MongoDB
+- Node.js v16.14.0 
+  - To install Node.js v16.14.0 through NVM (Node Version Manager), follow these steps:
+    1. Open your terminal.
+
+    2. Install NVM if you haven't already. You can install NVM by following the instructions at [NVM GitHub](https://github.com/nvm-sh/nvm).
+
+    3. Once NVM is installed, run the following command to install Node.js v16.14.0:
+       
+
+        ```bash
+        nvm install 16.14.0
+        ```
+
+        ```bash
+        node -v
+        ```
+
+        ```bash
+        v16.14.0
+        ```
+     5. You can set Node.js v16.14.0 as your default version with the following command:
+        ```bash
+        nvm use 16.14.0
```

Comment:
This is not the command to set nvm as the default.
You should have this command to activate that version but then run ```nvm alias default 16``` afterwards to set it as default.

Reply:
I remove the line 5 and spell OS correctly